core/templates/services/context.service.spec.ts

```diff
@@ -64,6 +64,15 @@ describe('Context service', () => {
       expect(ecs.getQuestionPlayerIsManuallySet()).toEqual(false);
       expect(ecs.isInQuestionPlayerMode()).toEqual(false);
     });
+
+    it('should correctly check the page context', ()=> {
```

Comment:
Is this correct? It seems to only check if the page context is either player or editor.

Reply:
Done! Didn't notice earlier